packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -51,6 +51,7 @@ export default class HostItem extends Component {
     () => this.props.hvSupportedVersions,
     () => this.props.item,
     (supportedVersions, host) =>
+      // In case the xoa plugin is not reachable we don't want to trigger a warning in the UI
```

Comment:
So, what are we doing to avoid triggering a warning?

Reply:
To avoid the warning, we set the `_isMaintained` to `true`

Reply:
Ok, that's worth explaining in the comment, something like:

```
// If could not fetch the list of maintained versions, consider this host up to date
```